app/src/main/java/com/claraberriel/mvpparking/mvp/presenter/ReservationPresenter.java

```diff
@@ -13,8 +13,10 @@
 import java.util.Date;
 
 public class ReservationPresenter {
-    public ReservationView reservationView;
-    public ReservationModel reservationModel;
+    @VisibleForTesting
```

Comment:
@imachadoglobant I don't like this annotation here. We can keep it for now but I have to be honest!